game/models.py

```
@@ -225,7 +225,62 @@ def __str__(self):
+
+    last_practiced = models.DateTimeField(
+        auto_now=True
+    )
+
+    class Meta:
+        unique_together = (
+            "user",
+            "opening_name"
+        )
+
+    def __str__(self):
```

Ruff (0.15.17)

[warning] Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)